### DISCUSSION TITLE
does it work with *.mkd extension files?

### DISCUSSION THREAD

**Discussion by m-faith** [jakewvincent/mkdnflow.nvim#150] [General]
I'm trying to switch from vim+vimwiki to nvim+mkdnflow and I'm partial to having my wiki files with `.mkd` extension. Is this supported by mkdnflow? It seems to be forcing `.md`.

**Comment by jakewvincent** [comment]
Hi @m-faith, sorry this has gone unanswered for forever. The answer is that the plugin does not support this out of the box; you'd have to update the filetype extensions in your config (see the default config [here](https://github.com/jakewvincent/mkdnflow.nvim#%EF%B8%8F-configuration)). Do you remember if you tried this?
